test/unit/touch-enroll-specs.js

```diff
@@ -0,0 +1,169 @@
+// transpile:mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { exec as execNode } from 'child_process';
+import B from 'bluebird';
+import { getUserDefault, setUserDefault, setTouchEnrollKey, setTouchEnrollKeys, getTouchEnrollKeys, touchEnrollMenuKeys, 
+  restoreTouchEnrollShortcuts, NS_USER_KEY_EQUIVALENTS, TOUCH_ENROLL_KEY_CODE } from '../../lib/touch-enroll';
+
+const exec = B.promisify(execNode);
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('touch-enroll.js', () => {
+
+  describe('getUserDefaults(), setUserDefaults()', () => {
+    let domain = 'fakeDomain';
+
+  // Nuke the contents of the domain so it's fresh every test case
+    let clearFakeDomain = async function () {
```

Comment:
The defaults are applied to a non-existent domain called 'fakeDomain' and then when the tests are completed, the 'fakeDomain' is deleted so that the system is returned to it's original state. 

I don't know if there's still a risk associated with that or not, but if there is I suppose I could switch over to using stubs. Were you thinking I should stub `exec` to control the command line output.